### ISSUE TITLE
annotation vs notes

### ISSUE THREAD

**Issue by cdanielmachado** [opencobra/memote#213]
**memote** reports missing annotations for metabolites and reactions when they are annotated using the <notes> element rather than the <annotation> element.

This should maybe be marked as a warning or "recommended practice" for the modeler to use the <annotation> element, rather than an error. 

The <notes> field was commonly used as the default way of storing annotations in the legacy cobra format.

**Comment by ChristianLieven** [comment]
I think this is a discussion worth having in the contest of the OpenCOBRA schema: https://github.com/opencobra/schema

**Comment by cdiener** [comment]
The common tone from the SBML contributors in that thread was that notes for annotations are actually wrong from the view of the SBML specification. Relevant excerpt from @draeger:

> Notes are not the right place to store any information that is relevant for computational tools. Notes in SBML are like comments in programing languages. You don't want to parse information out from there. This was done in the past and caused a lot of problems. People are still trying to write converters to get rid of repurposed notes in SBML.

**Comment by cdanielmachado** [comment]
Yes, I agree that the annotations field is a better practice than the notes field. Hence my initial comment, memote could report a warning, saying that it's preferable to use the annotations field, if it finds the annotations in the notes field. I think that is more appropriate than reporting missing annotations as an error, when they exist somewhere in the model (people just don't agree on where to put them).

**Comment by ChristianLieven** [comment]
I think this test is definitely very valid! However, I'd like to coordinate this a bit more with an overhaul to the cobrapy SBML parsers so that I know what to expect from the contents of the annotation and note fields in general. I have a feeling that not everything that is in the model may be adequately represented in the cobrapy model object yet.
